stbt-run

```diff
@@ -25,6 +25,14 @@ parser.add_argument(
     '--save-trace', metavar='FILE', default=None,
     help='Write state to this file, as xz compressed newline-seperated JSON')
 parser.add_argument(
+    '--write-screenshot', default='on-failure',
+    choices=['always', 'on-failure', 'never'],
+    help="Record a screenshot at the end of the test to screenshot.png")
+parser.add_argument(
+    '--write-thumbnail', default='never',
+    choices=['always', 'on-failure', 'never'],
+    help="Record a thumbnail at the end of the test to thumbnail.jpg")
```

Comment:
Did you choose "write" instead of "save" (like the existing "save-trace" and "save-video" on purpose?